### DISCUSSION TITLE
Center a form_card

### DISCUSSION THREAD

**Discussion by 5675sp** [h2oai/wave#828] [Ideas] [answered]
Is there a way to place a form_card in the middle of the page (or this will be a future implementation)?

**Comment by mtanco** [comment, marked as answer]
You can use the `align` parameter of the flexible layout zones. Here's an example:
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/4793561/119061996-86048280-b98a-11eb-81e7-1b0b85d27557.png">


```from h2o_wave import main, app, Q, ui


@app('/')
async def serve(q: Q):
    q.page['meta'] = ui.meta_card(
        box='',
        layouts=[
            ui.layout(
                breakpoint='xs',
                width='1200px',
                zones=[
                    ui.zone('header'),
                    ui.zone('body', align='center'),
                    ui.zone('footer'),
                ]
            )
        ]
    )
    q.page['header'] = ui.header_card(
        box='header',
        title='Center Form Card',
        subtitle='Learning application',
    )
    q.page['body'] = ui.form_card(
        box=ui.box('body', width='50%'),
        items=[
            ui.text_xl('Centered')
        ]
    )
    q.page['footer'] = ui.footer_card(
        box='footer',
        caption='Made with 💛 using [H2O Wave](https://github.com/h2oai/wave)!'
    )
    await q.page.save()
```

**Reply by mtanco** [reply]
The `justify` allows for center as well, along the other axis. In this case we can set the height of the zone to be larger than the card, and then get a centered card:
`ui.zone('body', align='center', justify='center', size='500px'),`
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/4793561/119062253-28246a80-b98b-11eb-87ae-38ab8cae3d92.png">

**Reply by 5675sp** [reply]
Thank you @mtanco
